CMakeLists.txt

```diff
@@ -71,6 +71,7 @@ option(OCOS_ENABLE_BERT_TOKENIZER "Enable the BertTokenizer building" ON)
 option(OCOS_ENABLE_BLINGFIRE "Enable operators depending on the Blingfire library" ON)
 option(OCOS_ENABLE_MATH "Enable math tensor operators building" ON)
 option(OCOS_ENABLE_DLIB "Enable operators like Inverse depending on DLIB" ON)
+option(OCOS_ENABLE_VENDOR_IMAGE_CODECS "Enable and use vendor image codecs if supported over libpng & libjpeg" ON)
 option(OCOS_ENABLE_OPENCV_CODECS "Enable cv2 and vision operators that require opencv imgcodecs." ON)
 option(OCOS_ENABLE_CV2 "Enable the operators in `operators/cv2`" ON)
```

Comment:
If I understand this correctly, we can disable both `OPENCV_CODECS` and `CV2` in GenAI. Because we only use image processor related code. No custom operators injection is needed by GenAI. @baijumeswani Could you be kind enough to confirm this?

Reply:
After you integrate the last ort-extensions main branch into GenAI, all of them opencv related flags should be turn off.  i.e. OCOS_ENABLE_OPENCV_CODECS OCOS_ENABLE_CV2 OCOS_ENABLE_VISION

Reply:
I decided to turn it off for now and only enable it with GenAI build. I think we can turn it on with future opencv removal work.